Sources/Apodini/Environment/Environment.swift

```diff
@@ -0,0 +1,60 @@
+/// A key for accessing values in the environment.
+public protocol EnvironmentKey {
+    /// Represents the type of the environment key’s value.
+    associatedtype Value
+    /// The default value of the `EnvironmentKey`.
+    static var defaultValue: Self.Value { get }
+}
+
+/// A collection of environment values.
+/// Custom environment values can be created by extending this struct with new properties.
+public struct EnvironmentValues {
+    var values: [ObjectIdentifier: Any] = [:]
+    
+    /// Initializer of `EnvironmentValues`.
+    public init() { }
+    
+    /// Accesses the environment value associated with a custom key.
+    public subscript<K>(key: K.Type) -> K.Value where K: EnvironmentKey {
+        get {
+            if let value = values[ObjectIdentifier(key)] as? K.Value {
+                return value
+            }
+            return K.defaultValue
+        }
+        set {
+            values[ObjectIdentifier(key)] = newValue
+        }
+    }
+    
+    /// Represents contents of the environment values instance.
+    public var description: String {
```

Comment:
I would suggest to remove this property as it is not needed and does not provide any additional value